pulp_2to3_migration/app/pre_migration.py

```diff
@@ -417,15 +417,17 @@ def pre_migrate_repo(record, repo_id_to_type):
         # if it was marked as such because it was not present in the migration plan
         repo.not_in_plan = False
         # check if there were any changes since last time
-        if last_unit_added != repo.pulp2_last_unit_added or \
-           last_unit_removed != repo.pulp2_last_unit_removed:
+        is_changed = (last_unit_added != repo.pulp2_last_unit_added
+                      or last_unit_removed != repo.pulp2_last_unit_removed)
+        if is_changed:
             repo.pulp2_last_unit_added = last_unit_added
             repo.last_unit_removed = last_unit_removed
             repo.pulp2_description = record.description
             repo.is_migrated = False
         repo.save()
 
-    return repo
+    if created or is_changed:
+        return repo
```

Comment:
It affects this https://github.com/pulp/pulp-2to3-migration/pull/249/files#diff-cb7ce2efbf29dbc7f500034642028cb9L385-L386.

I wonder if it makes sense to move the repocontent pre-migration here.